src/ir/context.rs

```diff
@@ -630,15 +634,29 @@ impl<'ctx> BindgenContext<'ctx> {
                 // argument names don't matter in the global context.
                 if (declaration.kind() == CXCursor_ClassTemplate ||
                     declaration.kind() ==
-                    CXCursor_ClassTemplatePartialSpecialization) &&
+                    CXCursor_ClassTemplatePartialSpecialization ||
+                    declaration.kind() == CXCursor_TypeAliasTemplateDecl) &&
```

Comment:
Yup, I agree, but we special-case ClassTemplate/ClassTemplatePartialSpecialization in a few places, but TypeAliasTemplateDecl only here. I'll try to organize it better.